test/integration/users.test.js

```diff
@@ -1040,4 +1037,61 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    beforeEach(async function () {
+      superUserId = await addUser(superUser);
+      superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+    });
```

Comment:
`cleanDb` already runs after each `it()` block as described here
https://github.com/Real-Dev-Squad/website-backend/blob/f61ff709973a586dda17eacaa0b9fef805de72e4/test/integration/users.test.js#L39-L41

Reply:
Okay I'll remove that from my describle block.